docs/content/how-to/disaster-recovery/automated-etcd-backup.md

```
@@ -0,0 +1,402 @@
+
+2. Fix the underlying issue (storage permissions, network access, bucket configuration)
+
+3. Delete the failed Job:
+
+    ```bash
+    oc delete job etcd-restore -n <hcp-namespace>
+    ```
+
+4. Remove the `EtcdSnapshotRestored` condition to allow the CPO to re-create the Job:
+
+    ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)